### PR TITLE
fix: WEB-178 Removed the default grey th background

### PR DESCRIPTION
**Pull request by Zougouda**

### CODE REVIEW

**Comment by PiiXiieeS** [comment]
@Zougouda this solution is not complete because default grey brackground used in Confluence tables will be now default to white background.

**Review by PiiXiieeS — CHANGES_REQUESTED** [review]
We new a more complete check like if no `data-highlight-colour` attribute provided then default to grey, otherwise not apply style so it will take the colour provided in the attribute.

This should display in purple:

```
<th data-highlight-colour="#c0b6f2" class="confluenceTh">
  <p style="text-align: center;">
    <strong>Digital IA &amp; FO Summer Tech Days</strong>
  </p>
</th>
```

while this should default to grey and not to white like in the current proposal:
```
<th class="confluenceTh">
  <p style="text-align: center;">
    <strong>Pros</strong>
  </p>
</th>
```

something like:
```
table.confluenceTable th.confluenceTh:not([data-highlight-colour]),
table.confluenceTable th.confluenceTh:not([data-highlight-colour]) > p {
  background-color: var(--highlight-grey);
}
```

File: src/assets/scss/_tables.scss [hunk 1]
```diff
@@ -86,8 +86,6 @@ table.confluenceTable th.confluenceTh {
 }
 
 /* Background colors */
```
Comment (PiiXiieeS) [inline src/assets/scss/_tables.scss]: table.confluenceTable th.confluenceTh:not([data-highlight-colour]),
table.confluenceTable th.confluenceTh:not([data-highlight-colour]) > p {
  background-color: var(--highlight-grey);
}

**Review by PiiXiieeS — DISMISSED** [review]

**Comment by PiiXiieeS** [comment]
@Zougouda I have committed the changes, take a look and confirm.

**Comment by Zougouda** [comment]
I approve your changes @PiiXiieeS and took the opportunity to make these css rules a little easier to read/maintain by exploiting the scss syntax a bit better.

**Pull request merged**